packages/manager/modules/billing/src/autoRenew/autorenew.html

```diff
@@ -56,6 +56,11 @@
             data-translate="billing_autorenew_service_renew_requires_mean_and_date"
         ></p>
         <p data-ng-if="$ctrl.defaultPaymentMean && !nicRenew.active">
+            <span
+                data-ng-if="$ctrl.choiceRenewDayTooltipAvailable"
+                data-translate="billing_autorenew_service_activate_alert_renew2016"
+                class="d-block"
+            ></span>
```

Comment:
**nitpick:** Avoid `span` with `d-block`. Use `p` instead

Reply:
In fact we are in <p>, I don't think is good <p> inside <p> ? and if i do this, i will to have also a class but class=mb-0

Reply:
It fixed, i put <br /> to replace class d-block